### PR TITLE
Make markdown refs compatible with prettier

### PR DESCRIPTION
**Pull request by ssbarnea**
Ensures that all our references keys are using lowercase in order to workaround known prettier bug.

This is also sorting another issue with editing markdown in vscode itself, which seems to not recognize or highlight the single `[foo]` blocks but works well on full ones like `[Foo][x]` and even inter-linking them.

Related: https://github.com/prettier/prettier/issues/3835
Required-By: #177

### CODE REVIEW

**Review by tomaciazek — APPROVED** [review]
LGTM!

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Community Code of Conduct
 
-Please see the official [Ansible Community Code of Conduct][CoC].
+Please see the official [Ansible Community Code of Conduct][coc].
 
-[CoC]:
+[coc]:
```
Comment (webknjaz) [inline .github/CODE_OF_CONDUCT.md:5]: I asked you not to do the lowercasing in another PR :man_facepalming: Why did you sneak it in anyway?
